src/state/StateServer.cpp

```diff
@@ -7,208 +7,277 @@
 #include <faabric/util/logging.h>
```

Comment:
Another grim diff, see the raw source [here](https://github.com/faasm/faabric/blob/zeromq/src/state/StateServer.cpp).